nexus/db-model/src/vni.rs

```diff
@@ -20,6 +20,12 @@ use serde::Serialize;
 #[diesel(sql_type = sql_types::Int4)]
 pub struct Vni(pub external::Vni);
 
+impl Vni {
+    pub fn random_system() -> Self {
+        Self(external::Vni::random_system())
```

Comment:
Collisions are unlikely but still possible using this - you could generate a random VNI with a CRDB query that also validates that it's not in use

Reply:
This function is just used to seed an initial choice. I did reuse `InsertVpcQuery` & `NextVni` for the DB query.